scripts/tests/chiptest/__init__.py

```diff
@@ -53,13 +74,34 @@ def tests_with_command(chip_tool: str, is_manual: bool):
         )
 
 
-def AllTests(chip_tool: str):
+# TODO We will move away from hardcoded list of yamltests to run yamltests parser/runner reaches
+# functionality parity with the code gen version.
+def _hardcoded_python_yaml_tests():
+    currently_supported_yaml_tests = ["TestConstraints.yaml"]
+
+    for name in currently_supported_yaml_tests:
+        yaml_test_path = _FindYamlTestPath(name)
+        if not yaml_test_path:
+            continue
```

Comment:
Log a "not found" message here for debugging? We seem to expect to always find all supported tests.

Reply:
Or maybe even raise exception of "... not found"